### PR TITLE
Update plot_permutation_importance.rst.txt

### PR DESCRIPTION
**Pull request by Sidakdeep23**
The warning "FixedFormatter should only be used together with FixedLocator" has been cleared by using mticker.

### CODE REVIEW

File: dev/_sources/auto_examples/inspection/plot_permutation_importance.rst.txt [hunk 1]
```diff
@@ -48,6 +48,7 @@ can mitigate those limitations.
 
     print(__doc__)
     import matplotlib.pyplot as plt
+    import matplotlib.ticker as mticker
```
Comment (Sidakdeep23) [inline dev/_sources/auto_examples/inspection/plot_permutation_importance.rst.txt:51]: importing mticker

**Comment by adrinjalali** [comment]
Hi, you should fix the issues on the scikit-learn repo, and not on the website repo directly.

**Comment by Sidakdeep23** [comment]
Oh sorry i didnt know that.

**Comment by GaelVaroquaux** [comment]
Should we try to use github's "Moderation" features to prevent pull
requests by mistake on this repo?

**Comment by adrinjalali** [comment]
The traffic is pretty low though. I'm easy either way.

**Pull request closed without merging**